MastodonSDK/Sources/CoreDataStack/Entity/Mastodon/Status.swift

```diff
@@ -99,6 +99,8 @@ public final class Status: NSManagedObject {
     @NSManaged public private(set) var deletedAt: Date?
     // sourcery: autoUpdatableObject
     @NSManaged public private(set) var revealedAt: Date?
+    
+    @Published public var translatedContent: String?
```

Comment:
Emmmmm. I'm not sure it is a good idea. If you want a property bound to an object. Please check Core Data `transient` property. The `Status.isSensitiveToggled` is an example. 

Also, the transient property lives in the same managed object context. So you always want to update and save in the `main` context only.

Reply:
Let me explain. The Core Data object is not magic. The different core data object maybe represent the same `entity` (the OOP concept here is a matter).  There is no guarantee for reused cells using the previous object. Also, maybe the same object will be used between different timelines. e.g. home and thread. 

Using `transient` property on the object will sync the state between different locations. It's will be a help.

Reply:
You're right! This totally makes sense. I'll fix this.

Reply:
Thank you @MainasuK done with https://github.com/mastodon/mastodon-ios/pull/741/commits/855f626c42fbb843090fc821a1b41830c6b63a9b